test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -643,3 +643,80 @@ Describe "Invoke-RestMethod tests" -Tags "Feature" {
         $result.Error | Should BeNullOrEmpty
     }
 }
+
+Describe "Bug fixes" -Tags "Feature" {
+
+    Context "InFile parameter negative tests" {
+
+        $testCases = @(
+            @{
+                Name = 'Validate error for Invoke-WebRequest -InFile ""'
+                ScriptBlock = {Invoke-WebRequest -Uri http://httpbin.org/post -Method Post -InFile ""}
+                ExpectedFullyQualifiedErrorId = 'WebCmdletInFileNotFilePathException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand'
+            }
+
+            @{
+                Name = 'Validate error for Invoke-WebRequest -InFile'
+                ScriptBlock = {Invoke-WebRequest -Uri http://httpbin.org/post -Method Post -InFile}
+                ExpectedFullyQualifiedErrorId = 'MissingArgument,Microsoft.PowerShell.Commands.InvokeWebRequestCommand'
+            }
+
+            @{
+                Name = "Validate error for Invoke-WebRequest -InFile  $TestDrive\content.txt"
+                ScriptBlock = {Invoke-WebRequest -Uri http://httpbin.org/post -Method Post -InFile  $TestDrive\content.txt}
+                ExpectedFullyQualifiedErrorId = 'PathNotFound,Microsoft.PowerShell.Commands.InvokeWebRequestCommand'
+            }
+
+            @{
+                Name = "Validate error for Invoke-RestMethod -InFile ''"
+                ScriptBlock = {Invoke-RestMethod -Uri http://httpbin.org/post -Method Post -InFile ''}
+                ExpectedFullyQualifiedErrorId = 'WebCmdletInFileNotFilePathException,Microsoft.PowerShell.Commands.InvokeRestMethodCommand'
+            }
+
+            @{
+                Name = "Validate error for Invoke-RestMethod -InFile <null>"
+                ScriptBlock = {Invoke-RestMethod -Uri http://httpbin.org/post -Method Post -InFile}
+                ExpectedFullyQualifiedErrorId = 'MissingArgument,Microsoft.PowerShell.Commands.InvokeRestMethodCommand'
+            }
+
+            @{
+                Name = "Validate error for Invoke-RestMethod -InFile  $TestDrive\content.txt"
+                ScriptBlock = {Invoke-RestMethod -Uri http://httpbin.org/post -Method Post -InFile $TestDrive\content.txt}
+                ExpectedFullyQualifiedErrorId = 'PathNotFound,Microsoft.PowerShell.Commands.InvokeRestMethodCommand'
+            }
+        )
+
+        It "<Name>" -TestCases $testCases {
+            param ($scriptblock, $expectedFullyQualifiedErrorId)
+
+            try
+            {
+                & $scriptblock
+                throw "CodeExecuted"
+            }
+            catch
+            {
+                $_.FullyQualifiedErrorId | should be $ExpectedFullyQualifiedErrorId
+            }
+        }
+    }
+
+    Context "InFile parameter positive tests" {
```

Comment:
>Context [](start = 4, length = 7)

and here - if httpbin.org does go down, these will fail.

I'm wondering whether we need, as part of the framework, our own webservice endpoint. 
This is _not_ something that I'm looking for in this PR, but something I think we should track. If we find we see a lot of failures because httpbin.org is down, we may need to do it.